### ISSUE TITLE
Where's the spec content?

### ISSUE THREAD

**Issue by xfq** [w3c/contentEditable#7]
There doesn't seem to be any content in https://w3c.github.io/contentEditable/

Is the work stopped? Or has it been moved to another place?

**Comment by saschanaz** [comment]
It was an effort to remove any unimplemented features. AFAIK there is no intent to implement from browsers, so no immediate plan to add anything back. https://github.com/w3c/contentEditable/pull/6 is served as a backup of the previous state.

**Comment by johanneswilm** [comment]
@xfq @saschanaz I believe there was some confusion as to what the point of this document was. Initially the document was started by Ben Peters of Microsoft as a proposal for new, hitherto not implemented features to achieve certain goals. I continued along those lines, adding/modifying according to consensus in the editing taskforce.

Unfortunately we are now part of a new working group in which it is no longer permitted to keep things in the main branch of a repository unless there are guarantees on implementations from browser makers. Given that in the case of this document this was not how the document had developed initially and that we had put it on the backburner while we were figuring out some other things related to Input Events, @marcoscaceres decided to remove all the contents for the time being. 

Please correct me if any of the above is incorrect.

**Comment by jyasskin** [comment]
The [charter](https://www.w3.org/2019/05/webapps-charter.html#working-mode:~:text=support%20from%20two%20or%20more%20web,can%20be%20made%20to%20a%20specification) says, 

> support from two or more web platform implementers is required before a substantive change can be made to a specification

I'm not seeing a requirement that two implementers must intend to implement or support a spec in order for it to be hosted in webapps, or even for it to be adopted by webapps, just that changes must improve things from the point of view of 2/3 of the implementers. Am I missing some charter text, or an email discussion where consensus was reached to delete specs that don't have multi-implementer support yet?

@marcoscaceres' https://github.com/w3c/contentEditable/commit/ae454fc84c784cd507a1e2f9a226d01f7ac519d4 that removed everything doesn't even have the "public record" of "support for a change/fix/feature" required by the charter.

**Comment by marcoscaceres** [comment]
Yeah, sorry, I dropped the ball here. Our plan was to figure out what was implemented or supported by 2 implementations AND to not duplicate what is already in HTML. I still think it's a worthy goal to get us to a point were we offload whatever we can to HTML and have spec that reflects reality.
